### PR TITLE
Add Project and Images Section

### PR DESCRIPTION
**Pull request by areetaw**
For style, instead of the previous forest theme, the site has a light purple theme with a more Googley font. In terms of content, there is now a functional `navigation bar` and a longer `about me` description, and `project` section filled with card objects from Bootstrap. After that, there is a carousel of images that allows the user to switch between images.

Screenshot 2020-06-23 at 9 10 03 PM

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,59 +1,293 @@
...
-            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-                quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-                Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
-                fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
-                culpa qui officia deserunt mollit anim id est laborum.
-            </p>
+			<div class="jumbotron text-left" style="color: #000000;">
+				<div class="mx-5 mx-xl-5">
+					<h2>hobbies</h2>
```
Comment (mrboxtobox) [inline portfolio/src/main/webapp/index.html:275]: Maybe fix the indentation here

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,62 +1,83 @@
...
+    font-size: 18px;
 }
 
 h1 {
-    margin-top: 1rem;
+    margin: 1rem;
     text-align: center;
 }
 
+img{
```
Comment (mrboxtobox) [inline portfolio/src/main/webapp/style.css:15]: You can add a space, `img {`

Comment (areetaw) [reply portfolio/src/main/webapp/style.css:15]: Thank you, fixed!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (areetaw) [reply portfolio/src/main/webapp/index.html:275]: Also, fixed!

**Pull request merged**